examples/web/src/features/memo/browser/styles.css

```
@@ -1,8 +1,12 @@
+.memo-surface {
     * { box-sizing: border-box; margin: 0; padding: 0; }
-    body {
+    & {
+      box-sizing: border-box;
+      margin: 0;
       font-family: 'Inter', 'Helvetica Neue', sans-serif;
       background: #1a1a2e;
       color: #e0e0e0;
+      min-height: 100vh;
       padding: 24px;
     }
```

Stylelint (17.14.0)

[error] Redundant nested style rule (block-no-redundant-nested-style-rules)

(block-no-redundant-nested-style-rules)